[WIP] Add completion for PowerShell (#1413)
arduino/arduino-cli: docs/command-line-completion.md

Remember to open a new shell to test the functionality.

### Powershell

Use `arduino-cli completion powershell > arduino-cli.ps1` to generate a temporary completion file. At this point you need to add
the content of the generated file to your PowerShell profile file.

1. `Get-Content -Path arduino-cli.ps1 | Add-Content -Path $profile` or add it by hand with your favourite text editor.

> 1. Move to first two "`using ...`" lines of `arduino-cli.ps1` on the top of the `$profile` file.

Reviewer: ```suggestion
1. The previous command added two `using namespace` lines, move them on top of the `$profile` file.
```

I think this was is clearer.
